dotcom-rendering/src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -324,7 +324,7 @@ const RemoteBanner = ({
 		window
 			.guardianPolyfilledImport(module.url)
 			.then((bannerModule: { [key: string]: JSX.Element }) => {
-				setBanner(() => bannerModule[module.name]); // useState requires functions to be wrapped
+				setBanner(() => bannerModule[module.name] ?? null); // useState requires functions to be wrapped
```

Comment:
@guardian/tx-engineers you might want to have a peek.

Reply:
I think this looks good! 👍

Strictly speaking this wasn't owned by TX, we looked after the similar `BrazeBanner.tsx` which sits along side this. Though now we've merged with Marketing Tools maybe this is us :-) /cc @tomrf1 in case you want to take a look.